doc/whats_new.rst

```diff
@@ -196,6 +196,10 @@ Bug fixes
       :clas:`discriminant_analysis.LinearDiscriminantAnalysis` now returns
       correct results. By `JPFrancoia`_
 
+    - Fixed bug in :class:`linear_model.SquaredEpsilonInsensitive`
```

Comment:
I made a mistake, it should be linear_model.sgd_fast.SquaredEpsilonInsensitive.

For the test part, although there are several tests for loss functions, they didn't assert the value of loss, just checked the final score.

I think we can add this test code below, which is not only for SquaredEpsilonInsensitive but for all loss functions in sgd. What do you think about this?
```python
from sklearn.linear_model.sgd_fast import SquaredEpsilonInsensitive
#......
def test_loss_functions():
    # Test loss functions
    # Test SquaredEpsilonInsensitive
    loss = SquaredEpsilonInsensitive(0.01)
    assert_equal(loss.dloss(1, 0.5), 0.98)
    assert_equal(loss.dloss(0.5, 1), -0.98)
    assert_equal(loss.dloss(1, 1), 0.) # where value is incorrect for current version
```